packages/beacon-node/src/network/core/networkCoreWorker.ts

```diff
@@ -120,9 +120,9 @@ wireEventsOnWorkerThread<ReqRespBridgeEventData>(
 );
 
 const libp2pWorkerApi: NetworkWorkerApi = {
-  close: () => {
+  close: async () => {
```

Comment:
It's always better to await a promise instead of returning promise from a non-async function. As we don't know the library implementation may be handling based on `AsyncFunction` constructor type. If we return promise from a non-async function it's constructor type remains `Function`. This could lead to some weird edge cases.